db/migrate/20241007150227_create_saved_searches.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSavedSearches < ActiveRecord::Migration[6.1]
+  def change
+    create_table :saved_searches do |t|
+      t.string :search_name
```

Comment:
I think search_name and search_description should probably just be name and description since there's already context given to it based on the model name e.g. saved_searches.name instead of saved_searches.search_name. It just reads a bit redundant and makes it a bit trickier to use.

```suggestion
      t.string :name
```